### PR TITLE
exit setFills routine cleanly if instance has no fills

### PR DESCRIPTION
**Pull request by joeyyang**
This PR makes the following change:
- prevents Toggler fatal runtime error if an instance has an empty set of fills

### CODE REVIEW

**Review by gblakeman — APPROVED** [review]
🛳💥

**Review by lynnerang — APPROVED** [review]
LGTM!

File: toggler/src/App.ts [hunk 1]
```diff
@@ -224,6 +224,8 @@ export default class App {
     const setFill = (instance: ToggleNode, newFill: RGBA): void => {
       // look up this fill from a GET request for team styles library at some point?
       const fills = cloneFills(instance.fills);
+      if (fills?.length < 1) return null;
```
Comment (lynnerang) [inline toggler/src/App.ts:227]: just realized something after I hit approve - if `fills` is `undefined` and causes the length to be `undefined`, does this evaluate to `false` when you'd want it to be `true`?  wondering if you might want something for all falseys like `if (!fills?.length)`.  could be wrong though, I get mixed up on JS math with non-numericals sometimes lol

Comment (joeyyang) [reply toggler/src/App.ts:227]: this is a great point - want to err on the side of explicit-ness - how about `if (!fills || fills.length < 1)`?

Comment (lynnerang) [reply toggler/src/App.ts:227]: I diggit!

**Pull request merged**